cypress-shared/support/common/support.js

```diff
@@ -150,33 +151,6 @@ export function fillAddress(postalCode) {
   })
 }
 
-function fillContactInfo() {
```

Comment:
Can you please revert this change, In commands.js we only have Cypress Commands
and this is a function it should not be in commands.js